nmostesting/suites/IS0702Test.py

```diff
@@ -513,6 +669,79 @@ def get_websocket_connection_sources(self, test):
                                             connection_sources[connection_uri].append(self.is04_sources[source_id])
         return connection_sources
 
+    def get_mqtt_broker_senders(self, test):
+        """Returns a dictionary of MQTT senders available for connection"""
+        broker_senders = {}
+
+        if len(self.is07_sources) > 0:
+            for source_id in self.is07_sources:
+                if source_id in self.sources_to_test:
+                    for sender_id in self.senders_to_test:
+                        flow_id = self.senders_to_test[sender_id]["flow_id"]
+                        if flow_id in self.is04_flows:
+                            if source_id == self.is04_flows[flow_id]["source_id"]:
+                                found_sender = self.senders_to_test[sender_id]
+                                if found_sender["transport"] == "urn:x-nmos:transport:mqtt":
+                                    if sender_id in self.senders_active:
+                                        if not self.senders_active[sender_id]["master_enable"]:
+                                            valid, response = self.is05_utils.perform_activation("sender", sender_id,
+                                                                                                 masterEnable=True)
+                                            if valid:
+                                                valid, response = self.update_active_sender(sender_id)
+                                            if not valid:
```

Comment:
```suggestion
                                            else:
```

Reply:
I was going to say "oh yes", but then I realised that the call to `update_active_sender` may make `valid` = `False`, so I think this code is correct.

Reply:
Ok, yes I see that now.